src/main/scala/za/co/absa/abris/avro/sql/AvroDataToCatalyst.scala

```diff
@@ -35,10 +36,10 @@ import scala.util.control.NonFatal
 import scala.util.{Failure, Success, Try}
 
 private[abris] case class AvroDataToCatalyst(
-  child: Expression,
-  abrisConfig: Map[String,Any],
-  schemaRegistryConf: Option[Map[String,String]]
-) extends UnaryExpression with ExpectsInputTypes {
+                                              child: Expression,
+                                              abrisConfig: Map[String, Any],
+                                              schemaRegistryConf: Option[Map[String, String]]
+                                            ) extends UnaryExpression with ExpectsInputTypes with Logging {
```

Comment:
Also, please don't change the indentation style. 

For class fields, we use the Scala stylistic guidelines
https://docs.scala-lang.org/style/declarations.html#classes

Reply:
>Do you need the Logging trait?

I have limited knowledge of Spark internals, but having looked at `org.apache.spark.internal.Logging` I believe the class considers support of Spark's distributed nature, more specifically:
```
  // Make the log field transient so that objects with Logging can
  // be serialized and used on another machine
  @transient private var log_ : Logger = null
```
Taking advantage of this train ensures our logging behaves in line with that of Spark.

Alternatively, we could take the more generic path:
```
private val logger = LoggerFactory.getLogger(this.getClass)
```

----

>For class fields, we use the Scala stylistic guidelines

Would you consider adding https://github.com/scalastyle/scalastyle-maven-plugin to the default Maven profile? Running the check with `./scalastyle-config.xml` I got 51 Scalastyle violations.

Adding the check to the CI pipeline would ensure style consistency.

Reply:
About the `Logging` trait: I meant if you need it in this specific class, since nothing is logged here.

Reply:
@gintautassulskus-elsevier We've added the scalastyle-maven-plugin in the meantime to the master branch.

Reply:
>About the Logging trait: I meant if you need it in this specific class, since nothing is logged here.

Oops, looked at the wrong class by mistake. Removed.